pkg/validate/validate_test.go

```diff
@@ -0,0 +1,58 @@
+package validate
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestGenerateExcludedRules(t *testing.T) {
+	type testCase struct {
+		name     string
+		input    string
+		expected []string
+	}
+	testCases := []testCase{
```

Comment:
As mentioned in https://github.com/FUSAKLA/promruval/pull/89/files#r1790443219

if we agree on the behaviour please add a test covering `check-test-label,      check test label` resulting to `check-test-label,check test label`

Reply:
there was one like that there already but updated it to as requested https://github.com/FUSAKLA/promruval/pull/89/files#diff-9746d4a5b310fcfb5b0e9bf8802c8d7bd7390bae0b5ef6b158efb6c64ed4c8b8R36